includes/batch.inc

```diff
@@ -117,9 +110,10 @@ class IslandoraPagedContentPDFBatchParentObject extends IslandoraScanBatchObject
   public function batchProcess() {
     module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
     $file = file_load($this->objectInfo['pdf_file']);
+    $this->label = $this->getTitle();
+    $this->getMods();
+    $this->getDc();
     islandora_paged_content_add_pdf_to_object($this, $file, TRUE);
-    // Use object_info to create some datastreams.
-    $this->label = $file->filename;
```

Comment:
The baseName is used in the event no XML is present. https://github.com/Islandora/islandora_batch/blob/7.x/includes/islandora_scan_batch.inc#L453